boot/bootutil/src/loader.c

```diff
@@ -515,7 +515,7 @@ split_image_check(struct image_header *app_hdr,
                   struct image_header *loader_hdr,
                   const struct flash_area *loader_fap)
 {
-    static void *tmpbuf;
+    static void *tmpbuf = NULL;
```

Comment:
static local variable is automatically initialized to 0 (so NULL).

Reply:
Indeed. Thank you! I'll wait before push more changes though, just in case this finally isn't a leak.

Reply:
Actually the `split_go()` functionality is not used by TF-M. We have just modified the code (adding FIH_CALL macros) to not break the build.

Do we know is there any user of this `split_go`  feature?

Reply:
What I see is that split_go() was something used in the mynewt bootloader https://github.com/apache/mynewt-core/blob/2673c6c414f05028abf12039033d3ec27f87d98b/boot/split/src/split.c#L66
Looks like a fossil here.
@utzig I think it can be removed form mcuboot. Can you confirm?

Reply:
No, I don't think it can. But I can ask on the Mynewt ML and deprecate it for next version (if there are no opposition). I would suggest not changing that code in your PRs; if it is broken and nobody fixes it, then nobody is really using it, otherwise users can send fixes.